cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+Cypress.Commands.add('fillNewEmployeeForm',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="office"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkDataDoesntExist',
+  (fullName, salary) => {
+    cy.get('tbody').contains(fullName).should('not.exist');
+    cy.get('tbody').contains(salary).should('not.exist');
+  });
+
+Cypress.Commands.add('compareRowValuesAfterSort', (employeeName, rowValue) => {
```

Comment:
or you can rename this into expectedValues (since rowValue means too many things already)